diff --git a/node_modules/react-native-image-base64/android/build.gradle b/node_modules/react-native-image-base64/android/build.gradle
index 162083b..35005f9 100644
--- a/node_modules/react-native-image-base64/android/build.gradle
+++ b/node_modules/react-native-image-base64/android/build.gradle
@@ -4,7 +4,7 @@ def safeExtGet(prop, fallback) {
 
 buildscript {
     repositories {
-        jcenter()
+        mavenCentral()
         google()
         maven {
             url 'https://maven.google.com'
@@ -32,9 +32,7 @@ android {
 
 repositories {
     mavenCentral()
-    maven {
-        url 'https://maven.google.com'
-    }
+    google()
 }
 
 dependencies {
